packages/core/src/Resource/Collection.js

```diff
@@ -27,14 +30,17 @@ function collectionLoad(collection) {
         data: {
           items: response.data,
         },
-        metadata: Object.assign({}, collection.state.metadata, { paging: response.paging }),
-      });
-    }).finally(() => {
-      mutateState(collection.stateStack, {
         metadata: Object.assign({}, collection.state.metadata, {
           loading: false,
           loaded: true,
           lastParameterBagState: collection.parameterBag.stateId,
+          paging: response.paging ? response.paging : collection.state.metadata.paging,
```

Comment:
heeft javascript geen ?? :P

Reply:
someString || "Cookies!";

Reply:
Wel, zou dan wel || zijn alleen is deze null-coalese niet goed genoeg in deze. Dit checkt enkel null. Deze check doet undefined en null